R/label.R

```diff
@@ -0,0 +1,62 @@
+#' A function to get the labels of data frame columns
+#'
+#' @param data a data frame
+#'
+#' @return labels of the input data frame
+#'
+#' @examples 
+#' get_label(r2rtf::r2rtf_adae)
+#' 
```

Comment:
```suggestion
#' 
#' @export
```

to address CMD issue